modules/openapi-generator/src/main/resources/lumen/Authenticate.php

```diff
@@ -1,11 +1,5 @@
 <?php
 
-/**
```

Comment:
Personally I prefer to keep this for "growth hacking" so that users will know that these files are generated by OpenAPI Generator.

Reply:
not a problem.  I will recall the headlines.